setup.py

```diff
@@ -2,13 +2,14 @@
 import platform
 import sys
 
-from setuptools import setup, find_packages, Extension
-from setuptools.command.test import test as TestCommand
-
 # ORDER MATTERS
 # Import this after setuptools or it will fail
```

Comment:
This is probably why we don't isort 😆

Reply:
oh, I cannot read.... I wonder if I can `# noqa` them 🤔